packages/modules/src/modules/banners/momentTemplate/components/MomentTemplateBannerBody.tsx

```diff
@@ -14,7 +14,7 @@ import { BannerRenderedContent } from '../../common/types';
 interface MomentTemplateBannerBodyProps {
     mainContent: BannerRenderedContent;
     mobileContent: BannerRenderedContent;
-    highlightedTextSettings: HighlightedTextSettings;
+    highlightedTextSettings: HighlightedTextSettings | undefined;
```

Comment:
Should we mark optional properties with a `?`, for example `highlightedTextSettings?: HighlightedTextSettings;`?

Reply:
With the current approach we'd always need to pass a value for `highlightedTextSettings` which is what we want here I reckon. It could have the value undefined but the prop always has to be present. The alternative would incorrectly imply that you can optionally add that prop when using the component